packages/firebase_remote_config/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.2.0+3
+
+* Auto retrieve pubspec version to report usage to
+  Firebase.
```

Comment:
Slightly more formal here, and also I don't think you need the extra linebreak.

```suggestion:-2+1
* Automatically use version from pubspec.yaml when reporting usage to Firebase.
```

Reply:
Done